koku/api/migrations/0013_auto_20200226_1953.py

```diff
@@ -8,16 +8,7 @@
 
 def reload_cost_model_metric_map(apps, schema_editor):
     """Load cost model metrics."""
-    CostModelMetricsMap = apps.get_model("api", "CostModelMetricsMap")
-    CostModelMetricsMap.objects.all().delete()
-
-    data = pkgutil.get_data("api", "metrics/data/cost_models_metric_map.json")
-
-    data = json.loads(data)
-
-    for entry in data:
-        map = CostModelMetricsMap(**entry)
-        map.save()
+    pass
```

Comment:
@adberglund  Can you take a look?

If we are going to make this change it makes more sense to me to delete the method and the *RunPython* call:
https://github.com/project-koku/koku/pull/1986/commits/f1b23c08e09f91f7630f82e7fbacbc78b6aeeda9#diff-6776daf81424264056a8d93f5f0a077dR26

Reply:
@chambridge I updated the PR.

I deleted the pre-existing method and RunPython call.